s27125-HubertJozwiak/zajecia1/instalacja_debian_linux.txt

```diff
@@ -0,0 +1,535 @@
+- `FAT32` - system plików FAT32, który jest przeznaczony dla małych partycji
+i urządzeń przenośnych.
+- `Swap area` - partycja swap, która służy jako przestrzeń wymiany, gdy
+brakuje pamięci RAM.
+- `Physical volume for encryption` - partycja, która służy jako wolumen do
+szyfrowania danych.
+- `Physical volume for LVM` - partycja, która służy jako wolumen do
+zarządzania logicznymi wolumenami.
+- `Phusical volume for RAID` - partycja, która służy jako wolumen do
+zarządzania macierzami RAID.
+- `Do not use the partition` - partycja, która nie jest używana w systemie.
+
+52. Resztę opcji pozostawiamy domyślnie i naciskamy klawisz `Done setting up
+the partition`.
+Opcje mount options:
+- `discard` - pozwala na automatyczne usuwanie nieużywanych bloków z partycji.
+- `noatime` - pozwala na wyłączenie zapisu czasu dostępu do plików, co może
+przyspieszyć operacje odczytu i zapisu.
+- `nodiratime` - pozwala na wyłączenie zapisu czasu dostępu do katalogów, co
+może przyspieszyć operacje odczytu i zapisu.
+- `relatime` - pozwala na zapis czasu dostępu do plików tylko w przypadku
+zmiany, co może przyspieszyć operacje odczytu i zapisu.
+- `nodev` - pozwala na wyłączenie dostępu do urządzeń w partycji.
+- `noexec` - pozwala na wyłączenie wykonywania plików w partycji.
+- `ro` - pozwala na zamontowanie partycji jako tylko do odczytu.
+- `sync` - pozwala na synchroniczne zapisy do partycji, co może spowolnić
+operacje zapisu.
+- `usrquota` - pozwala na ustawienie limitów dyskowych dla użytkowników.
+- `grpquota` - pozwala na ustawienie limitów dyskowych dla grup.
+- `user_xattr` - pozwala na dodawanie dodatkowych atrybutów do plików.
+
+53. Następnie utworzymy partycję, na której bedą foldery domowe
+użytkowników. Wybieramy opcję `Create a new partition` i naciskamy klawisz
+`Enter`.
+
+54. Tak jak wcześniej, instalator poprosi nas o podanie rozmiaru partycji. W
+naszym przypadku przydzielamy 7.5 GB.
+
+55. Następnie instalator poprosi nas o wybór typu partycji. Wybieramy opcję
+`Logical` i naciskamy klawisz `Enter`.
+
+56. Następnie instalator poprosi nas o wybór miejsca, w którym chcemy
+umieścić partycję. Wybieramy opcję `Beginning` i naciskamy klawisz `Enter`.
+
+57. W następnym kroku ustawiamy opcję `Mount point` na `/home` i naciskamy
+klawisz `Done setting up the partition`.
+
+58. Ostatnim krokiem jest utworzenie głównej partycji systemowej zamontowanej
+w `/`. Wybieramy opcję `Create a new partition` i naciskamy klawisz `Enter`.
+
+59. Następnie instalator poprosi nas o podanie rozmiaru partycji. W naszym
+przypadku przydzielamy resztę dostępnej przestrzeni.
+
+60. Następnie instalator poprosi nas o wybór typu partycji. Wybieramy opcję
+`Primary` i naciskamy klawisz `Enter`.
+
+61. Następnie instalator poprosi nas o wybór miejsca, w którym chcemy
+umieścić partycję. Wybieramy opcję `Beginning` i naciskamy klawisz `Enter`.
+
+62. W następnym kroku pozostawiamy opcję `Mount point` jako `/` i naciskamy
+`Done setting up the partition`.

```

Comment:
Brak opisu bootable flag(i)
Co to jest?
Kiedy należy ją zaznaczyć a kiedy nie?
Na co ma to wpływ?